index-handlers/src/test/java/no/sikt/nva/nvi/index/OpenSearchClientTest.java

```diff
@@ -138,11 +143,11 @@ void shouldReturnDocumentsFromIndexAccordingToGivenOffsetAndSize() throws IOExce
 
     @Test
     void shouldReturnEverythingIfYearFilterAndAffiliationsAreNotSet() throws IOException, InterruptedException {
```

Comment:
Confused by the test name and searchParameters in the test. The test is called `shouldReturnEverythingIfYearFilterAndAffiliationsAreNotSet`, but you are adding affiliations as search params in the test?
Rename test?